spec/factories/recipients_shououts.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
```

Comment:
Factories need to be fixed

Reply:
Thank @rogergraves  for your question about duplicates. I refactored code and the code got very nice